classes/controller/FrontController.php

```diff
@@ -119,7 +119,7 @@ class FrontControllerCore extends Controller
     /** @var bool If true, forces display to maintenance page. */
     protected $maintenance = false;
 
-    /** @var string[] Adds excluded $_GET keys for redirection */
+    /** @var string[] Adds excluded _GET keys for redirection */
```

Comment:
```suggestion
    /** @var string[] Adds excluded $_GET keys for redirection */
```

Reply:
yes, but CS fixer does not like it, what should I do? :)